src/main/app/src/app/formulieren/formulier/formulier.component.ts

```diff
@@ -94,11 +120,31 @@ export class FormulierComponent implements OnInit {
 
   opslaan() {
     this.bezigMetOpslaan = true;
-    this.submit.emit(this.formGroup.value);
+    this.formPartial.emit(this.formGroup.value);
+    this.bezigMetOpslaan = false;
```

Comment:
I don't understand why 2 lines ahead the boolean gets set to false again; is this correct?

Reply:
Yes it is correct. The "bezigMetOpslaan" boolean disables the form submit button during the time the forms is submitted and saved. And it should be enabled again afterwards.